packages/aws-cdk/lib/init.ts

```diff
@@ -187,8 +183,15 @@ async function assertIsEmptyDirectory() {
 async function initializeGitRepository() {
     if (await isInGitRepository(process.cwd())) { return false; }
     print('Initializing a new git repository...');
-    await execute('git', 'init');
-    return true;
+    try {
+        await execute('git', 'init');
+        await execute('git', 'add', '.');
+        await execute('git', 'commit', '--message="Initial commit"', '--no-gpg-sign');
+        return true;
+    } catch (e) {
+        error('Error initializing git repository; you will have to do this by hand.');
```

Comment:
warning (it doesn't mean cdk-init failed). I would also relax the wording: `Unable to initialize a git repository for your project: ${reason}`)

Reply:
Reason (from git) has already been printed to stderr. Nothing to print here, really.